README.md

```diff
@@ -34,7 +34,10 @@ Install it:
 npm i --save-dev eslint-doc-generator
 ```
 
-Add scripts to `package.json` (both a lint script to ensure everything is up-to-date in CI and an update script) (add any config options in the `update:eslint-docs` script):
+Add scripts to `package.json`:
+
+- Both a lint script to ensure everything is up-to-date in CI and an update script for contributors to run locally
+- Add any [config options](#configuration-options) in the `update:eslint-docs` script only
```

Comment:
```suggestion
- Add any [configuration options](#configuration-options) in the `update:eslint-docs` script only
```

Reply:
We're a bit inconsistent about this wording, but in most of this readme, we've been using "config", so I think I'll leave as is for now.